docs/source/reference/config.rst

```diff
@@ -185,6 +185,14 @@ Available fields and semantics:
     #         - "*": my-default-security-group
     security_group_name: my-security-group
 
+    # Encrypted boot disk (optional).
+    #
+    # Set to true to encrypt the boot disk of all AWS instances launched by
+    # SkyPilot. This is useful for compliance with data protection regulations.
+    #
+    # Default: false.
+    disk_encrypted: true
```

Comment:
Starting to think having opposite value in the comment & actual is unintuitive. Maybe write the default value on L194?

Reply:
Good point! Changed to default value instead.